src/task_10_2.py

```diff
@@ -0,0 +1,14 @@
+# Создать текстовый файл и записать в него 6 строк.
+# Записываемые строки вводятся с клавиатуры
+
+
+f = open('../../test.txt', 'w')
+n = input()
+v = input()
+c = input()
+x = input()
+z = input()
+l = input()
```

Comment:
Лучше все input занести в цикл, и сделать его range(N)

```suggestion
for _ in range(N):
    f.write(input() + '\n')
```